cms/templates/js/unit-access-editor.underscore

```diff
@@ -0,0 +1,133 @@
+deletedGroups.forEach(function (fullId) {
+    var schemeId = fullId.split('-')[0];
+    var groupId = fullId.split('-')[1];
+    var partitionExists = false;
+    selectablePartitions.forEach(function (partition) {
+        if (partition['id'] === schemeId) {
+            partitionExists = true;
+            partition['groups'].push(
+                {
+                    "deleted": true,
+                    "selected": true,
+                    "id": groupId,
+                    "name": ""
+                }
+            );
+        }
+    })
+
+    if (!partitionExists) {
+        selectablePartitions.push(
+            {
+                "scheme": "cohort",
+                "id": schemeId,
+                "groups": [
+                    {
+                        "deleted": true,
+                        "selected": true,
+                        "id": groupId,
+                        "name": "Deleted Group"
+                    }
+                ],
+                "name": "Content Group Configuration"
+            }
+        )
+    }
+})
+%>
+<form>
+<h3 class="modal-section-title">
+    <%- gettext('Unit Access') %>
+</h3>
+<div class="modal-section-content access-change">
+    <label for="partition-select" class="group-select-title"><%- gettext('Restrict access to:') %>
+        <select class="user-partition-select" id="partition-select">
+            <option value="-1" selected ="selected"><%- gettext('Select a group type') %></option>
+            <% for (var i=0; i < selectablePartitions.length; i++) { %>
+                <option id="<%- selectablePartitions[i].id %>-option" value="<%- selectablePartitions[i].id %>"><%- selectablePartitions[i].name %></option>
+            <% } %>
+        </select>
+    </label>
+    <br>
+    <% if (selectablePartitions.length > 0) { %>
+        <div class="user-partition-group-checkboxes">
+            <% for (var i=0; i < selectablePartitions.length; i++) { %>
+                <div id="<%- selectablePartitions[i].id %>-checkboxes">
+                    <div class="partition-group-directions">
+                        <%- gettext('Select one or more groups:') %>
+                        <% for (var j = 0; j < selectablePartitions[i].groups.length; j++) { %>
+                            <div class="field partition-group-control">
+                                <input type="checkbox" id="content-group-<%- selectablePartitions[i].groups[j].id %>" value="<%- selectablePartitions[i].groups[j].id %>" class="input input-checkbox"
+                                    <% // Checkbox is checked if the partition exists and the specific partition's group
+                                       // is in the xblock's group access, or if the group has been deleted. %>
+                                    <% if ((groupAccess[selectablePartitions[i].id]
+                                            && groupAccess[selectablePartitions[i].id].indexOf(selectablePartitions[i].groups[j].id) !== -1)
```

Comment:
I'm confused about the data structure here. Is it not possible to pass through the data structure that already has the "selected" value (I'm looking here at the similar code in visibility_editor.html)?